config/locales/fil/business_info.fil.yml

```diff
@@ -2,21 +2,21 @@ fil:
   business_info:
     title: "Refuge Restrooms Business Info"
```

Comment:
Thank you for the changes!

Sorry I missed this earlier, but is there an appropriate translation for "business info"?

\- DeeDeeG

Reply:
I've made a translation for this but it's not direct translation, just the same concept of this word. Also used this in `about.fil.yml` wherein there is also a `Business FAQ` word. Thanks for noticing!